website/content/en/docs/best-practices/best-practices.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Operator Best Practices"
+linkTitle: "Best practices"
+weight: 1
+description: This guide describes the best practices concepts to write operators.
+---
+
+## Development
+
+Considerations for operator developers:
+
+- An Operator should manage a single type of application, essentially following the UNIX principle: do one thing and do it well.
+
+- If an application consists of multiple different tiers or components, multiple Operators should be written for each of them. If the application for example consists of Redis, AMQ and MySQL, there should be 3 Operators, not one.
+
+- If there is significant orchestration and sequencing involved, an Operator should be written that represents the entire stack, in turn delegating to other Operators for orchestrating their part of it.
+
+- Operators should own a CRD and only one Operator should control a CRD on cluster. Two Operators managing the same CRD calls for trouble. In case where an API exists but with multiple implementations, we typically have a no-op Operator (without any deployment/reconciliation loop) that defines that shared API and other Operators depend on this Operator to provide one implementation of the API, e.g. similar to PVCs or Ingress.
+
+- Inside an Operator, multiple controllers should be used if multiple CRDs are managed. This helps with separation of concerns and code readability. Note that this doesn't necessarily mean one container image per controller, but rather one reconciliation loop (which could be running as part of the same operator binary) per CRD.
+
+- An operator shouldn't deploy or manage other operators (such patterns are known as meta or super operators). It's the Operator Lifecycle Manager's job to manage the deployment and lifecycle of operators.
+
+- If multiple operators must be packaged/shipped as a single entity (via the same CSV), then add all owned and required CRDs, as well as all deployments for operators that manage the owned CRDs, to the same CSV.
+
+- Writing an Operator involves using the Kubernetes API and there is always the same boilerplate code required to connect and interact with it. Use a framework like Operator-SDK to save yourself time with this and get a suite of tooling to ease development and testing.
```

Comment:
```suggestion
- Writing an Operator involves using the Kubernetes API and as such it will always have the same boilerplate code required to connect and interact with it. Use a framework like the Operator-SDK to save yourself time with this and get a suite of tooling to ease development and testing.
```